pkg/datarepair/checker/checker.go

```diff
@@ -251,30 +251,32 @@ func (checker *Checker) updateSegmentStatus(ctx context.Context, pointer *pb.Poi
 	return nil
 }
 
-// IrreparableProcess picks items from irreparabledb and add them to the repair
-// worker queue if they, now, can be repaired.
+// IrreparableProcess iterates over all items in the irreparabledb. If an item can
+// now be repaired then it is added to a worker queue.
```

Comment:
nit -- sp/now/not

Reply:
are you saying to replace "now" with "not"?

Reply:
depends, I read it as `If an item cannot be repaired then it is added to a worker queue` but looking at the original comment you can ignore me